boards/nordic/nrf54h20dk/nrf54h20dk_nrf54h20_cpurad_0_9_0.yaml

```diff
@@ -1,8 +1,8 @@
 # Copyright (c) 2024 Nordic Semiconductor ASA
 # SPDX-License-Identifier: Apache-2.0
 
-identifier: nrf54h20dk/nrf54h20/cpurad
-name: nRF54H20-DK-nRF54H20-Radio (revision 0.9.0)
```

Comment:
We might want to wait with adding cpurad/iron, because we don't support it in any way yet. Also, if we add this, shouldn't it be an additional board file like was added for app? We also don't have any iron variants of ppr/flpr (again I think they could be added later)